staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -84,6 +86,15 @@ var UpdateNodeSpecBackoff = wait.Backoff{
 	Jitter:   1.0,
 }
 
+const (
+	// NodeSuccessSynced is used as part of the Event 'reason' when a Node is synced
+	NodeSuccessSynced = "Synced"
```

Comment:
This constant and `NodeMessageResourceSynced` is only used once so we probably don't need constants and can just use the string literal directly in the event message.